python-sdk/tutorial.ipynb

```diff
@@ -8,22 +8,20 @@
     "\n",
     "Welcome to the nuScenes tutorial. Make sure you have the nuScenes DB schema available.\n",
     "\n",
-    "This demo assumes the database itself is available at `/data/nuscenes`"
+    "This demo assumes the database itself is available at `/data/sets/nuscenes`"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "# Let's start by initializing the database\n",
     "%matplotlib inline\n",
     "from nuscenes.nuscenes import NuScenes\n",
     "\n",
-    "nusc = NuScenes(version='v0.1', dataroot='/data/nuscenes', verbose=True)"
+    "nusc = NuScenes(version='v1.0', dataroot='/data/sets/nuscenes', verbose=True)"
```

Comment:
`v1.0-mini`?

Reply:
Yes please. That one reverted back somehow.

Reply:
`v1.0-mini` would be the best default.